snippets/computed.cson

```diff
@@ -0,0 +1,146 @@
+'.source.coffee':
+
+
+
+
+  'Ember Snippets: computed properties':
+    'prefix': 'computed--help'
+    'body':
+      """
+      ${1:
```

Comment:
I just feel they would be hard to maintain, and I don't think it's necessary. I'm so-so on the issue, If you feel that it would be important then lets keep them.